packages/sku/src/lib/language-utils.ts

```diff
@@ -37,22 +36,15 @@ export function getValidLanguagesForRoute(route) {
 
 const LANGUAGE_NAMED_PARAM = 'language';
 
-/**
- * @param {string} pathname
- * @param {string} route
- */
-function getLanguageParamFromUrl(pathname, route) {
+function getLanguageParamFromUrl(pathname: string, route: any) {
   const match = routeMatcher(route)(pathname);
 
-  return match ? match.params[LANGUAGE_NAMED_PARAM] : null;
+  return match
+    ? (match.params as Record<string, any>)[LANGUAGE_NAMED_PARAM]
```

Comment:
Would prefer to avoid this `any` type creeping into the codebase. Passing a generic parameter to `match` seems to work well enough:
```ts
  return match<Record<string, string>>(normalisedRoute);
```

https://github.com/seek-oss/sku/blob/065bd6543dc3fa697f6052d79f83e03303420972/packages/sku/src/lib/routeMatcher.ts#L16